modules/revcontentBidAdapter.js

```diff
@@ -0,0 +1,290 @@
+// jshint esversion: 6, es3: false, node: true
+'use strict';
+
+import {registerBidder} from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import {ajax} from '../src/ajax';
+
+const BIDDER_CODE = 'revcontent';
+const NATIVE_PARAMS = {
+  title: {
+    id: 0,
+    name: 'title'
+  },
+  image: {
+    id: 3,
+    type: 3,
+    name: 'img'
+  },
+  sponsoredBy: {
+    id: 5,
+    name: 'data',
+    type: 1
+  }
+};
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: ['banner', 'native', 'video'],
```

Comment:
seems like we are only supporting `native` as of now since `nativeParams` is required on every bid?